sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/e2e/application-exception.institution.getException.e2e-spec.ts

```diff
@@ -0,0 +1,165 @@
+import { saveFakeApplicationWithApplicationException } from "../application-exception-helper";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitutionLocation,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { saveStudentApplicationForCollegeC } from "../../../student/_tests_/e2e/student.institutions.utils";
+
+describe("ApplicationExceptionInstitutionsController(e2e)-getException", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeF: Institution;
+  let collegeFLocation: InstitutionLocation;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation(collegeF);
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+  });
+
+  it(
+    "Should get the eligible application exception of a student belonging to a public " +
+      "institution when the same public institution requests to see the details.",
+    async () => {
+      // Arrange
+
+      const application = await saveFakeApplicationWithApplicationException(
+        db.dataSource,
+        { institutionLocation: collegeFLocation },
+        { applicationExceptionStatus: ApplicationExceptionStatus.Approved },
+      );
+      const applicationException = application.applicationException;
+
+      const endpoint = `/institutions/application-exception/student/${application.student.id}/exception/${applicationException.id}`;
+      const institutionUserToken = await getInstitutionToken(
+        InstitutionTokenTypes.CollegeFUser,
+      );
+      // Act/Assert
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          exceptionStatus: applicationException.exceptionStatus,
+          submittedDate: applicationException.createdAt.toISOString(),
+          exceptionRequests: applicationException.exceptionRequests.map(
+            (request) => ({
+              exceptionName: request.exceptionName,
+            }),
+          ),
+        });
+    },
+  );
+
+  it("Should throw not found response error when the public institution requests to see the exception details that does not exist for the institution .", async () => {
+    // Arrange
+
+    const application = await saveFakeApplicationWithApplicationException(
+      db.dataSource,
+      { institutionLocation: collegeFLocation },
+      { applicationExceptionStatus: ApplicationExceptionStatus.Approved },
+    );
+
+    const endpoint = `/institutions/application-exception/student/${application.student.id}/exception/9999999`;
```

Comment:
As per the test case definition, the institution should not be able to see the exception for the application which does not belong to them. But we are checking for exceptionId that doesn't exist at all. I would suggest suggest to save one more application with exception for the same student for CollegeC and use that exception id to test.